packages/node/src/config.js

```diff
@@ -18,6 +18,17 @@ module.exports = {
   logger: {
     ...schema.logger,
     defaultValue: () => getPrefixedConsole()
+  },
+  onUnhandledError: {
+    defaultValue: () => (err, report, logger) => {
```

Comment:
is `report.request` guaranteed to be defined at this point?

Reply:
No, it could be `undefined`, but `report` is guaranteed to be defined, so accessing `report.request` is fine.